newspaper/article.py

```diff
@@ -190,24 +193,32 @@ def download(self, input_html=None, title=None, recursion_counter=0):
             parsed_url = urlparse(self.url)
             if parsed_url.scheme == "file":
                 html = self._parse_scheme_file(parsed_url.path)
+                # Hard code to text/html for now
+                mime_type = "text/html"
```

Comment:
Not sure I agree with hard coding this to "text/html"... Personally I would probably rather lean towards None or an empty string

Reply:
Yes, as mentioned in the PR, I don't think think this is correct either. If we don't want to be "smart" here, e.g. detect it somehow, then I guess `None` would be a good choice. Thoughts?

Reply:
I'd lean towards `None`, simply because I've seen too many web pages that are PDFs but do not have the ".pdf" extension. This is just one example off the top of my head but I'm sure there are others

Reply:
`None` it is. :)